### DISCUSSION TITLE
flow() function fails with `Cannot read properties of null (reading 'x')`

### DISCUSSION THREAD

**Discussion by egil** [naver/billboard.js#2546] [Q&A]
I have a simple one spline chart that I want to flow updated values into. However, when I call `flow()`, I get a `Cannot read properties of null (reading 'x')` error message. Any suggestions for how to do this correctly?

Here is my current attempt:

```js
// base css
import 'billboard.js/dist/theme/insight.css';
import bb from 'billboard.js';

const chart = bb.generate({
  data: {
    columns: [
      ["x", 20, 30, 40]
    ]
  },
});

chart.flow({
   columns: [
       ["x", 50, 60]
   ]
});
```

Link to try out the code: https://stackblitz.com/edit/zyjb8b?file=index.js

Thanks, Egil

**Comment by netil** [comment]
Hi @egil, it should be working without error. I'll be handling this in separate issue.
https://github.com/naver/billboard.js/issues/2595
